kratos/tests/cpp_tests/utilities/test_parallel_utilities.cpp

```diff
@@ -123,14 +123,24 @@ KRATOS_TEST_CASE_IN_SUITE(CustomReduction, KratosCoreFastSuite)
     KRATOS_CHECK_EQUAL(ReturnValueReducer.max_value, 0.0 );
     KRATOS_CHECK_EQUAL(ReturnValueReducer.max_abs, nsize-1 );
 
+    typedef CombinedReduction< SumReduction<double>,
+                               MinReduction<double>,
+                               MaxReduction<double>,
+                               SubReduction<double>
+            > MultipleReduction;
+
     auto combined = IndexPartition<unsigned int>(data_vector.size()).
-        for_reduce<CombinedReduction< SumReduction<double>, MinReduction<double>>>(
+        for_reduce<MultipleReduction>(
             [&](unsigned int i){
-                return std::pair<double,double>{data_vector[i],data_vector[i]}; //note that here the lambda returns the values to be reduced
+                return std::tuple<double,double,double,double>{
+                    data_vector[i],data_vector[i],data_vector[i],data_vector[i]}; //note that here the lambda returns the values to be reduced
```

Comment:
We could probably add another overload for the `CombinedReduction::operator()` so that it would work with scalar (or rather non-tuple) values. Then it would send the same value to each of the child reductors, and the above lambda would be
```cpp
[&](unsigned int i){ return data_vector[i]; }
```

Reply:
I have updated my example: https://gist.github.com/ddemidov/e8da1ff5786278e51571c28759dd602c